libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -68,6 +92,15 @@ void AP_Frsky_Telem::setup_wfq_scheduler(void)
     set_scheduler_entry(BATT_2, 1300, 500);     // 0x5008 Battery 2 status
     set_scheduler_entry(BATT_1, 1300, 500);     // 0x5008 Battery 1 status
     set_scheduler_entry(PARAM, 1700, 1000);     // 0x5007 parameters
+#if HAL_WITH_FRSKY_TELEM_BIDIRECTIONAL
+    set_scheduler_entry(MAV, 35, 25);           // mavlite
+    // initialize sport sensor IDs
+    set_sport_sensor_id(_frsky_parameters->_sport_uplink_id,_sport_config.uplink_sensor_id);
+    set_sport_sensor_id(_frsky_parameters->_sport_dnlink1_id,_sport_config.downlink1_sensor_id);
+    set_sport_sensor_id(_frsky_parameters->_sport_dnlink2_id,_sport_config.downlink2_sensor_id);
+    // initialize sport
+    hal.scheduler->register_io_process(FUNCTOR_BIND_MEMBER(&AP_Frsky_Telem::process_sport_rx_queue, void));
```

Comment:
Humph!  Definitely going to have to get my own thread for the dataflash logger ;-)

Reply:
Should I create my own thread to process incoming packets? I do not have strict time constraints here so I can probably lower the priority if required